tests/manage/pv_services/test_rbd_thick_provisioning.py

```diff
@@ -0,0 +1,119 @@
+import logging
+import pytest
+from concurrent.futures import ThreadPoolExecutor
+
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import (
+    skipif_ocs_version,
+    polarion_id,
+    ManageTest,
+    tier1,
+    acceptance,
+)
+from ocs_ci.helpers import helpers
+from ocs_ci.utility.utils import TimeoutSampler
+
+log = logging.getLogger(__name__)
+
+
+@skipif_ocs_version("<4.8")
+class TestRbdThickProvisioning(ManageTest):
+    """
+    Tests to verify PVC creation and consumption using RBD thick provisioning enabled storage class
+
+    """
+
+    @pytest.fixture(autouse=True)
+    def setup(self, storageclass_factory):
+        """
+        Create storage class
+
+        """
+
+        self.sc_obj = storageclass_factory(
+            interface=constants.CEPHBLOCKPOOL,
+            new_rbd_pool=True,
+            rbd_thick_provision=True,
+        )
+
+    @acceptance
+    @tier1
+    @polarion_id("OCS-2500")
+    def test_rbd_thick_provisioning(self, multi_pvc_factory, pod_factory):
+        """
+        Test to verify PVC creation and consumption using RBD thick provisioning enabled storage class
+
+        """
+        # Dict represents the PVC size and size of the file used to consume the volume
+        pvc_and_file_sizes = {1: "900M", 5: "4G"}
+
+        access_modes_rbd = [
+            constants.ACCESS_MODE_RWO,
+            f"{constants.ACCESS_MODE_RWO}-Block",
+            f"{constants.ACCESS_MODE_RWX}-Block",
+        ]
+
+        pvcs = []
+
+        # Create PVCs
+        for pvc_size in pvc_and_file_sizes.keys():
+            pvc_objs = multi_pvc_factory(
+                interface=constants.CEPHBLOCKPOOL,
+                storageclass=self.sc_obj,
+                size=pvc_size,
+                access_modes=access_modes_rbd,
+                status=constants.STATUS_BOUND,
+                num_of_pvc=3,
+                timeout=120,
+            )
+            for pvc_obj in pvc_objs:
+                pvc_obj.io_file_size = pvc_and_file_sizes[pvc_size]
+                pvc_obj.storage_type = (
+                    "block" if pvc_obj.get()["spec"]["volumeMode"] == "Block" else "fs"
```

Comment:
I see the words "block" and "fs" lots of time in the pv tests. I think we can replace "block" and "fs" with the constants `STORAGE_TYPE_BLOCK`, and `STORAGE_TYPE_FS` or similar names.

Reply:
Done